test/port-finder-socket.test.js

```diff
@@ -25,7 +25,8 @@ function createServers (callback) {
     function (next) {
       const server = net.createServer(function () { }),
             name = base === 0 ? 'test.sock' : 'test' + base + '.sock';
-      let sock = path.join(socketDir, name);
+      const socket = path.join(socketDir, name);
+      let sock = socket;
```

Comment:
Just so I understand - why are we changing this and what is going on - e.g., why are we setting another variable for the socket handle

Reply:
to run `fs.unlinkSync(socket);` on windows, is that correct? (and we just never did that prior to this?)

Reply:
related does calling `fs.openSync(sock, 'w');` with `socket` instead of `sock` work? (below, not part of diff)

Reply:
If the platform is windows, we append the pipe stuff to the front of `sock` to be passed to `server.listen`. However, we create the file without the pipe stuff, so that's why I've used the two separate variables, `socket` to track the filename as generated, and then `sock` to be modified with the pipe stuff.

Reply:
sound good